aslam_offline_calibration/kalibr/python/yaml_to_camera_info_publisher

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+import rospy
+import yaml
+from sensor_msgs.msg import CameraInfo
+
+def yaml_to_CameraInfo(yaml_fname):
+    """
+    Parse a yaml file containing camera calibration data (as produced by
+    rosrun camera_calibration cameracalibrator.py) into a
+    sensor_msgs/CameraInfo msg.
+
+    Parameters
+    ----------
+    yaml_fname : str
+        Path to yaml file containing camera calibration data
+
+    Returns
+    -------
+    camera_info_msg : sensor_msgs.msg.CameraInfo
+        A sensor_msgs.msg.CameraInfo message containing the camera calibration
+        data
+    """
+    # Load data from file
+    with open(yaml_fname, "r") as file_handle:
+        calib_data = yaml.load(file_handle)
+    # Parse
+    camera_info_msg = CameraInfo()
+    camera_info_msg.width = calib_data["image_width"]
+    camera_info_msg.height = calib_data["image_height"]
+    camera_info_msg.K = calib_data["camera_matrix"]["data"]
+    camera_info_msg.D = calib_data["distortion_coefficients"]["data"]
+    camera_info_msg.R = calib_data["rectification_matrix"]["data"]
+    camera_info_msg.P = calib_data["projection_matrix"]["data"]
+    camera_info_msg.distortion_model = calib_data["distortion_model"]
+    camera_info_msg.binning_x = 1
+    camera_info_msg.binning_y = 1
+    # camera_info_msg.
```

Comment:
Yeah. I was comparing ROS+OpenCV yaml to camera info against Kalibr Camchain params. Do not need this script anymore.